src/lib/users/ssh_authorized_keyring.rb

```diff
@@ -218,8 +218,13 @@ def create_ssh_dir(user, group)
       def write_file(owner, group)
         file = SSHAuthorizedKeysFile.new(authorized_keys_path)
         file.keys = keys
+
         log.info "Writing #{keys.size} keys in #{authorized_keys_path}"
-        file.save && FileUtils::Chown("#{owner}:#{group}", authorized_keys_path, false)
+
+        return unless file.save
+
+        FileUtils::Chown("#{owner}:#{group}", authorized_keys_path, false)
+        FileUtils::Chmod(AUTHORIZED_KEYS_PERMS, authorized_keys_path, false)
```

Comment:
Just a note (we already discussed it offline): is not needed at all: https://github.com/yast/yast-users/blob/7735c840cc98327f526a731e5ff87769d906b79b/src/lib/users/ssh_authorized_keys_file.rb#L119. Actually, the `AUTHORIZED_KEYS_PERMS` looks like a left-over.